megamek/i18n/megamek/client/messages.properties

```diff
@@ -3539,7 +3620,7 @@ HeatEffects.tsm.mt.49=-7 MP; +7 BTH
 HeatEffects.tsm.mt.50=Automatic shutdown
 
 #Loading Units into Bays
-LoadingBay.baytoomany=Too many units selected.\nThe selected bay has a remaining capacity of 
+LoadingBay.baytoomany=Too many units selected.\nThe selected bay has a remaining capacity of {0}{1}. 
```

Comment:
Just checking that this wasn't supposed to have a space?

Reply:
Probably...